app/views/play/level/tome/editor/snippets.coffee

```diff
@@ -172,7 +172,9 @@ module.exports = (SnippetManager, autoLineEndings) ->
 
     beginningOfLine = session.getLine(pos.row).substring(0,pos.column - prefix.length)
 
-    unless (fullPrefixParts.length < 3 and /^(hero|self|this|@|db|game|ui)$/.test(fullPrefixParts[0]) ) or /^\s*$/.test(beginningOfLine)
+    # we already returned if fullPrefixParts.length > 2, so fullPrefixParts.length < 3 always true here
+    # and we want to enable auto completion when cursor is inside a function call as param. so add more check
+    unless /^(hero|self|this|@|db|game|ui)$/.test(fullPrefixParts[0]) or /^\s*$/.test(beginningOfLine) or /(,| |\()$/.test(beginningOfLine)
```

Comment:
Should we get rid of old `self`, `this`, and `@`, now that it's been years since we have moved to `hero`, and add in `pet` here?